Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/MetsKitodoReader.java

```diff
@@ -64,10 +66,12 @@ static Mets readStringToMets(String xmlString) throws JAXBException {
      * @return The Mets object in mets-kitodo format.
      */
     static Mets readUriToMets(URI xmlFile) throws JAXBException, IOException {
-        if (Paths.get(xmlFile).toFile().exists()) {
+        FileManagementInterface fileManagementModule = new KitodoServiceLoader<FileManagementInterface>(
+                FileManagementInterface.class).loadModule();
+        if (fileManagementModule.fileExist(xmlFile)) {
             JAXBContext jaxbMetsContext = JAXBContext.newInstance(Mets.class);
             Unmarshaller jaxbUnmarshaller = jaxbMetsContext.createUnmarshaller();
-            return (Mets) jaxbUnmarshaller.unmarshal(xmlFile.toURL());
+            return (Mets) jaxbUnmarshaller.unmarshal(fileManagementModule.getFile(xmlFile));
```

Comment:
Here it should be the same: to use `FileManagement` to open the URI (since Java on its own may not know how to interpret it):

    try (InputStream inputStream = fileManagementModule.read(xmlFile)) {
        return (Mets) jaxbUnmarshaller.unmarshal(inputStream);
    }